js/src/layers/ImageService.js

```diff
@@ -0,0 +1,74 @@
+// Copyright (c) Jupyter Development Team.
+// Distributed under the terms of the Modified BSD License.
+
+const L = require('../leaflet.js');
+const layer = require('./Layer.js');
+const proj = require('../projections.js');
+
+export class LeafletImageServiceModel extends layer.LeafletLayerModel {
+  defaults() {
+    return {
+      ...super.defaults(),
+      _view_name: 'LeafletImageServiceView',
+      _model_name: 'LeafletImageServiceModel',
+      // image server url
+      url: '',
+      // response format
+      f: 'image',
+      // output image format
+      format: 'jpgpng',
+      // data type of the raster image
+      pixelType: 'UNKNOWN',
+      // pixel value or list of pixel values representing no data
+      noData: [],
+      // how to interpret no data values
+      noDataInterpretation: '',
+      // resampling process for interpolating the pixel values
+      interpolation: '',
+      // lossy quality for image compression
+      compressionQuality: '',
+      // order of bands to export for multiple band images
+      bandIds: [],
+      // time instance or extent for image
+      time: [],
+      // rules for rendering
+      renderingRule: {},
+      // rules for mosaicking
+      mosaicRule: {},
+      // image transparency
+      transparent: false,
+      // endpoint format for building the export image url
+      endpoint: '',
+      // image service attribution
+      attribution: '',
+      // coordinate reference system
+      crs: null,
+      // emit when clicked or hovered
+      interactive: false,
+      // update interval for panning
+      updateInterval: 200
```

Comment:
Hi @HaudinFlorence, great question. I wanted to put this PR together to address [ImageService layers that do not have layers available from other services](https://github.com/jupyter-widgets/ipyleaflet/issues/932). This was the case for digital elevation model products from the [Polar Geospatial Center (PGC)](https://www.pgc.umn.edu/data/), such as [REMA](https://elevation2.arcgis.com/arcgis/rest/services/Polar/AntarcticDEM/ImageServer) and [ArcticDEM](https://elevation2.arcgis.com/arcgis/rest/services/Polar/ArcticDEM/ImageServer). While ArcticDEM does have a WMS url for the gray hillshade layer, it doesn't for any other layer. REMA, unfortunately, is only available through ImageService queries. One of my colleagues suggested I just try to prompt PGC to add more services, but I thought as a first step it might be easier to add the same capabilities available through the DEM web applications to ipyleaflet.

Reply:
> This a detail point but from an homogeneity point of view, description of the attributes are only set on the python side, in the docstrings.

Great point. I removed these comments in 93394bf